### PR TITLE
all admin UI tables redone

### LINKED ISSUE #182: Sorting doesn't work as expected in Admin tables
Clicking the two sided arrow and title in the column headers on tables in the Admin does not work as expected.

When you click on the header, it sorts that column in ASC order. I would expect that when I click it again, it switches to DESC order. And if I clicked it a third time, it would sort by ASC order, etc.

Please fix this on all existing sortable table columns in the Admin and Dashboard.
- https://livepeer.com/app/admin/streams
- https://livepeer.com/app/admin/keys

### PR DESCRIPTION
**Pull request by darkdarkdragon**
**What does this pull request do? Explain your changes. (required)**
all admin UI tables redone
all now has paging, sorting and filtering
(admin users, streams, tokens and webhook tables)

**Does this pull request close any open issues?**
Fixes #182 #176 #179 

**Screenshots (optional):**
image

image

image

image

### CODE REVIEW

File: packages/api/src/controllers/api-token.js [hunk 1]
```diff
@@ -39,8 +44,19 @@ app.delete('/:id', authMiddleware({}), async (req, res) => {
   res.end()
 })
 
+const fieldsMap = {
+  id: `api_token.ID`,
+  name: `api_token.data->>'name'`,
+  lastSeen: `api_token.data->'lastSeen'`,
```
Comment (iameli) [inline packages/api/src/controllers/api-token.js:50]: Only one `>` on this one?

**Comment by wohlner** [comment]
@darkdarkdragon, @julianbenegas is going to review this PR. He will help guide it into production and incorporate the remaining work on this epic (https://app.zenhub.com/workspaces/basement-studio-6006e89b3591a0000ef02311/issues/livepeer/livepeer-com/180) as it makes sense.

**Comment by julianbenegas** [comment]
Sorting not working for [https://livepeer.monster/app/admin](https://livepeer.monster/app/admin)


https://user-images.githubusercontent.com/40034115/105847262-22f67680-5fbc-11eb-9e5f-b46a075b816f.mp4

**Comment by julianbenegas** [comment]
@darkdarkdragon I'm getting `unexpected error` on

- [https://livepeer.monster/app/admin/keys](https://livepeer.monster/app/admin/keys)
- [https://livepeer.monster/app/admin/streams](https://livepeer.monster/app/admin/streams)
- [https://livepeer.monster/app/admin/webhooks](https://livepeer.monster/app/admin/webhooks)

CleanShot 2021-01-26 at 09 53 42

**Comment by julianbenegas** [comment]
When copying an email (to filter by it), it doesn't work. It's because there's a weird character inserting itself after the "@"

https://user-images.githubusercontent.com/40034115/105849110-b6c94200-5fbe-11eb-8705-fa8dec088351.mp4

**Comment by julianbenegas** [comment]
Tested on livepeer.monster, but all bugs happen also on [https://livepeer-com-git-it-admin-keys.livepeerorg.now.sh/](https://livepeer-com-git-it-admin-keys.livepeerorg.now.sh/)

**Comment by darkdarkdragon** [comment]
@julianbenegas right now it doesn't work because backend changes was not deployed to the staging. Will try to fix this today.

**Review by iameli — COMMENTED** [review]
Dang it, I did this review like weeks ago and never actually hit "submit review"

File: packages/api/src/controllers/api-token.js (hunk 1 above)
Comment (darkdarkdragon) [reply packages/api/src/controllers/api-token.js:50]: yes, only on this one - it is integer field. `->>` converts it to string and that makes sorting incorrect

**Pull request merged**